README.md

```diff
@@ -10,7 +10,7 @@
 
 This repository provides [Docker](https://www.docker.com/) images of the [Rocq Prover](https://rocq-prover.org/).
 
-These images are based on [this parent image](https://hub.docker.com/r/rocq/base/), itself based on [Debian 12 Slim](https://hub.docker.com/_/debian/) and relying on [opam 2.0](https://opam.ocaml.org/doc/Manual.html):
+These images are based on [this parent image](https://hub.docker.com/r/rocq/base/), itself based on [Debian 12 Slim](https://hub.docker.com/_/debian/) and relying on [opam 2.3](https://opam.ocaml.org/doc/Manual.html):
```

Comment:
Indeed, thanks. I'm not sure it'll be useful to hardcode the major version here. Maybe something like this?

```suggestion
These images are based on [this parent image](https://hub.docker.com/r/rocq/base/), itself based on [Debian 12 Slim](https://hub.docker.com/_/debian/) and relying on the last version of [opam 2.x](https://opam.ocaml.org/doc/Manual.html):
```